src/Entity/OgMembership.php

```diff
@@ -320,6 +320,22 @@ public function preSave(EntityStorageInterface $storage) {
       throw new \LogicException('OG membership can not be created for an empty or anonymous user.');
     }
 
+    if (!$this->get('entity_id')->value) {
+      // Group was not set.
+      throw new \LogicException('Membership cannot be set for an empty or an unsaved group.');
```

Comment:
Don't think so. This system was built to support changes submitted programmatically like REST